zcncore/transaction_query_base.go

```diff
@@ -184,6 +188,117 @@ func (tq *TransactionQuery) randOne(ctx context.Context) (string, error) {
 	}
 }
 
+func (tq *TransactionQuery) FromConsensus(ctx context.Context, query string, handle QueryResultHandle) error {
+
+	min := getMinShardersVerify()
+
+	if err := tq.validate(min); err != nil {
```

Comment:
Not sure if this validate is necessary, as long as there are total sharders num > 0, we can send requests to them instead of returning error

Reply:
removed